terraform/environments/ci/files/decrypt.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+unset IFS
+out_path=${1:-/out/secrets.env}
+rm "$out_path" 2> /dev/null
```

Comment:
`rm -f` (so that `set -e` doesn't make it explode when `$out_path` doesn't exist yet).